### ISSUE TITLE
Favourite / Like feature

### ISSUE THREAD

**Issue by markheath** [NuGet/NuGetGallery#440]
I'd like to be able to mark interesting nuget packages as my favourites. This would allow the site to alert me to new releases (either by visiting a page, or emailing me, or even an RSS feed).

Other interesting features would also be possible, such as an option to just show my favourites in the VS Manage NuGet Packages dialog.  Or even an algorithm that can recommend other packages I might like based on what I have already marked as a favourite.

**Comment by evil-shrike** [comment]
definitely must have

**Comment by robertmuehsig** [comment]
Is this still a thing? Like the "star"/"watch" feature on GitHub? Would this something be "up-for-grabs"?

**Comment by anangaur** [comment]
I am not sure how helpful this feature is going to be given we already have the RSS feature ( 😄 😄 ). What are your thoughts on the scenarios it will light up?

**Comment by robertmuehsig** [comment]
Well... it would somehow be a copy of the "star" feature on GitHub - but obviously not all packages are OSS and hosted on GitHub, so it might be ok.
In theory you could have another metric for "interesting" packages from the community with this mechanic in place and build some clever recommendation based on the information ("... maybe you should check package X because you also liked package Y"). 
Not 100% sure if this would be super helpful for NuGet.org, but everytime I click on the [Package](https://www.nuget.org/packages) link the table seems to be very static and I'm not 100% sure why NUnit seems to be always on top. 
With a "star" feature you could maybe split between the "common package downloads" (e.g. all packages that are included in the default templates with millions of packages downloads) and community engagement.

To summarize: For a "pure" package site it's really not needed, but if you want to get more community engagement it might be worth it.

**Comment by anangaur** [comment]
Its interesting discussion to have considering we were having the same discussion just today within the team 😄

Coming to the benefits, I agree this could be a good metric to rank but then we do have plans to switch to "Installs" that will remove the auto restore counts or CI counts. The problem is non trivial as all installs in `PackageReference` are essentially `restores` but then if we can solve the problem of counting Installs and start looking at recent data (say last 6 months), the ranking problem would be solved.

From developer's use case perspective though, I see limited benefits (as of now, and hence want to have continued discussion to bring about more usecases) post favoriting a package through stars.

**Comment by anangaur** [comment]
From https://developercommunity.visualstudio.com/idea/620459/add-nuget-to-favorites-and-synchroniz-with-account.html:
Hope to provide a function to add a Nuget package to Favorites, and the Favorites List would be synchronized with my Account. Wherever I login VisualStudio via my Account, I can find Nuget I familiar with.
Thx.
